src/items/itemsLandingPage.component.tsx

```diff
@@ -0,0 +1,399 @@
+          width: '100%',
+          '@media print': {
+            display: 'none',
+          },
+        }}
+        item
+      >
+        <Button
+          startIcon={<ArrowBackIcon />}
+          component={Link}
+          to={
+            catalogueItemIdData && catalogueItemIdData.id
+              ? `/catalogue/item/${catalogueItemIdData.id}/items`
+              : '/catalogue'
+          }
+          sx={{ mx: 0.5 }}
+          variant="outlined"
+        >
+          {catalogueItemIdData
+            ? `Back to ${catalogueItemIdData.name} items table view`
+            : 'Home'}
+        </Button>
+
+        <Button
+          startIcon={<PrintIcon />}
+          sx={{ mx: 0.5 }}
+          variant="outlined"
+          onClick={() => {
+            window.print();
+          }}
+        >
+          Print
+        </Button>
+      </Grid>
+      {catalogueItemIdData && itemData && (
+        <Grid item xs={12}>
+          <Grid container spacing={1} flexDirection="column">
+            <Grid item xs={12}>
+              <Typography sx={{ margin: 1, textAlign: 'center' }} variant="h4">
+                {catalogueItemIdData.name}
+              </Typography>
+            </Grid>
+            <Grid item xs={12}>
+              <Typography sx={{ margin: 1, textAlign: 'center' }} variant="h6">
+                Description:
+              </Typography>
+              <Typography
+                sx={{ margin: 1, textAlign: 'center' }}
+                variant="body1"
+                color="text.secondary"
+              >
+                {catalogueItemIdData.description}
+              </Typography>
+            </Grid>
+
+            <Grid container spacing={1} sx={{ px: '192px' }}>
+              <Grid
+                item
+                xs={12}
+                onClick={toggleDetails}
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                  cursor: 'pointer',
+                }}
+                aria-label={`${showDetails ? 'Close' : 'Show'} item details`}
+              >
+                {showDetails ? (
+                  <>
+                    <Typography variant="h6">Details</Typography>
+                    <ExpandLessIcon />
+                  </>
+                ) : (
+                  <>
+                    <Typography variant="h6">Details</Typography>
+                    <ExpandMoreIcon />
+                  </>
+                )}
+              </Grid>
+
+              <Grid item xs={12}>
+                <Collapse in={showDetails}>
+                  <Grid container spacing={1}>
+                    <Grid item xs={12} sm={6} md={4}>
+                      <Typography align="left" color="text.primary">
+                        Serial Number
+                      </Typography>
+                      <Typography align="left" color="text.secondary">
+                        {itemData?.serial_number ?? 'None'}
```

Comment:
```suggestion
                        {itemData.serial_number ?? 'None'}
```
If above already checked this.